OnlineStore/OnlineStore.DAL/Repositories/interfaces/IOrderRepository.cs

```diff
@@ -0,0 +1,9 @@
+﻿using OnlineStore.Domain.Entities;
+
+namespace OnlineStore.DAL.Repositories.interfaces
+{
+    internal interface IOrderRepository : IRepository<Order>
```

Comment:
interfaces of repositories should have public access modificator. check everywhere